app/views/components/_share-links.html.erb

```diff
@@ -0,0 +1,28 @@
+<%
+  title ||= 'Share this page'
+  facebook_href ||= false
+  twitter_href ||= false
+%>
+<% if facebook_href || twitter_href %>
+<div class="app-c-share-links-wrapper">
```

Comment:
Can the outer class be `app-c-share-links` for consistency with other components?